packages/web/src/client/components/Video/VideoPlayer.tsx

```diff
@@ -134,3 +144,31 @@ const Video = styled('video', {
   width: '100%',
   height: '100%',
 })
+
+const Control = styled('button', {
+  bottom: '16px',
+  right: '16px',
+  background: 'white',
+  width: '40px',
+  height: '40px',
+  position: 'absolute',
+  display: 'flex',
+  borderRadius: '50%',
+  border: 'none',
+  '&:hover': {
+    background: '#e5e5e5',
+    cursor: 'pointer',
+  },
+  span: {
+    width: '24px',
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'relative',
+
+    img: {
+      position: 'absolute',
```

Comment:
Im confused about to styling decisions here, can you elaborate why you felt this to be necessary?